src/main/java/com/googlecode/download/maven/plugin/internal/checksum/Checksums.java

```diff
@@ -61,44 +61,44 @@ public boolean isValid(final File file) {
     }
 
     /**
-     * Validates the file with supplied signatures.
+     * Validates the file with supplied checksums.
      * @param file File to validate.
-     * @throws Exception If the file didn't match any supplied signature.
+     * @throws Exception If the file didn't match any supplied checksum.
      */
     public void validate(final File file) throws Exception {
-        for (final Map.Entry<Signature, String> entry : this.supplied.entrySet()) {
-            SignatureUtils.verifySignature(
+        for (final Map.Entry<Checksum, String> entry : this.supplied.entrySet()) {
+            ChecksumUtils.verifyChecksum(
                 file, entry.getValue(),
                 MessageDigest.getInstance(entry.getKey().algo())
             );
         }
     }
 
     /**
-     * Fill the map of signatures.
-     * @param md5 Supplied md5 signature, may be {@literal null}.
-     * @param sha1 Supplied sha1 signature, may be {@literal null}.
-     * @param sha256 Supplied sha256 signature, may be {@literal null}.
-     * @param sha512 Supplied sha512 signature, may be {@literal null}.
-     * @return A map of a signature type to a digest; {@literal null} digests
+     * Fill the map of checksums.
+     * @param md5 Supplied md5 checksum, may be {@literal null}.
+     * @param sha1 Supplied sha1 checksum, may be {@literal null}.
+     * @param sha256 Supplied sha256 checksum, may be {@literal null}.
+     * @param sha512 Supplied sha512 checksum, may be {@literal null}.
+     * @return A map of a checksum type to a digest; {@literal null} digests
      *  are not included.
      */
-    private static Map<Signature, String> create(
+    private static Map<Checksum, String> create(
         @Nullable final String md5, @Nullable final String sha1,
         @Nullable final String sha256, @Nullable final String sha512
     ) {
-        final Map<Signature, String> digests = new EnumMap<>(Signature.class);
+        final Map<Checksum, String> digests = new EnumMap<>( Checksum.class);
         if (md5 != null) {
-            digests.put(Signature.MD5, md5);
+            digests.put( Checksum.MD5, md5);
         }
         if (sha1 != null) {
-            digests.put(Signature.SHA1, sha1);
+            digests.put( Checksum.SHA1, sha1);
         }
         if (sha256 != null) {
-            digests.put(Signature.SHA256, sha256);
+            digests.put( Checksum.SHA256, sha256);
         }
         if (sha512 != null) {
-            digests.put(Signature.SHA512, sha512);
+            digests.put( Checksum.SHA512, sha512);
```

Comment:
```suggestion
            digests.put(Checksum.SHA512, sha512);
```

Reply:
LOl, actually in a BeanShell script in your repo I jus found that style of using braces with spaces, when doing a global search. But I am not going to fix those, because they are out of scope here. I just wanted to mention the inconsistency.

Reply:
Same goes for line 46 of `LoggingProgressReport::initiate`. I also leave that up to you.